src/AsyncWebSocket.cpp

```diff
@@ -829,7 +829,7 @@ void AsyncWebSocketClient::binary(AsyncWebSocketMessageBuffer * buffer)
 
 IPAddress AsyncWebSocketClient::remoteIP() {
     if(!_client) {
-        return IPAddress(0U);
+        return IPAddress((uint32_t)0);
```

Comment:
It'd be great to get this merged some day. There seems to be a whole history of people forking the lib to fix this one line!
I'm using current libs and modern platformio with a ESP32 chip and this error is a hard blocker for using this library at all.

The overload with no arguments defaults to a zero address, so I'd suggest leaving the address out altogether...
```suggestion
        return IPAddress();
```
Both work though, so 🤷